core/src/main/java/io/questdb/cairo/CairoException.java

```diff
@@ -208,6 +220,10 @@ public StackTraceElement[] getStackTrace() {
         return result;
     }
 
+    public boolean isApplyBlockError() {
```

Comment:
language: "apply block error" has a different meaning than "block apply error". I think the latter was intended.

Reply:
renamed as suggested [297fdc9](https://github.com/questdb/questdb/pull/5347/commits/297fdc93edf97602b1ef9d99c1502407a35df53e)